chaos-monkey-docs/src/main/asciidoc/endpoints.adoc

```diff
@@ -2,6 +2,8 @@
 == Spring Boot Actuator Endpoints ==
 Chaos Monkey for Spring Boot offers you some built in endpoints exposed via https://docs.spring.io/spring-boot/docs/current/reference/html/actuator.html#actuator.endpoints[JMX or HTTP]. This allows you to change configuration at runtime.
 
+https://docs.spring.io/spring-boot/docs/current/reference/html/actuator.html[Spring Boot Actuator] needs to be included in your project dependencies.
```

Comment:
This is a friendly hint, that you need Actuator when you want to use Actuator Endpoints right?

I was first a bit worried that all our users needs actuator even when they not use actuator endpoints :D

Reply:
Yes, exactly. I just added this as a hint, we could mark it as such with a `Hint:` prefix?

Reply:
I've made an suggestion below. I think Hint: is not needed